src/components/CurratedContentCard/index.tsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react'
+import styled from '@emotion/styled'
+import { Image } from '../Image'
+import {
+  color,
+  radius,
+  space,
+  fontSize,
+  fontWeight,
+  device,
+  lineHeight,
+} from '../../theme'
+import { SmallText } from '../Text'
+import { transition } from '../../../src'
+
+const Container = styled.div`
+  background-color: ${color.elevatedBackground};
+  display: flex;
+  inline-size: 302px;
+  block-size: 314px;
+  flex-direction: column;
+  cursor: pointer;
+  border-radius: ${radius.lg};
+  overflow: hidden;
+
+  @media ${device.tablet} {
+    inline-size: 639px;
+    block-size: 244px;
+    flex-direction: row;
+  }
+
+  @media ${device.laptop} {
+    &:hover {
+      img {
+        transition: all ${transition};
+        transform: scale(1.1);
+      }
+    }
+    &:not(:hover) {
+      img {
+        transition: all ${transition};
+        transform: scale(1);
+      }
+    }
+  }
+`
+const Content = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: flex-start;
+  padding: ${space[16]};
+  max-block-size: 127px;
+  max-inline-size: 302px;
```

Comment:
Instead of using sizes, maybe better to use a grid?

Reply:
Now using aspect-ratio, i think that is best